internal/api/http.go

```diff
@@ -187,8 +187,17 @@ func inspectableMIMEType(t string) bool {
 	return strings.HasPrefix(t, "text/") || jsonTypeRE.MatchString(t)
 }
 
-func isValidHost(reqHost, hrtHost string) bool {
-	return strings.Contains(reqHost, hrtHost)
+func isRequestHostValid(reqHost, hrtHost string) bool {
```

Comment:
Let's avoid acronyms and abbreviations in variable names when not necessary.
- `reqHost` could be `requestHost`
- `hrtHost` (what is "hrt" in this context anyway?) could simply be `domain`

Reply:
Yep can see where you are coming from here and have made the suggested changes.